### ISSUE TITLE
Open entities in new tab rather than replacing existing

### ISSUE THREAD

**Issue by LaylBongers** [OurMachinery/themachinery-public#701]
When opening an entity asset to edit it, I want to quickly navigate back to the scene I had open previously, and switch between it and the asset to make changes.

Currently when opening an entity it will replace the scene tab, but it would be nicer for this specifically if it were to open in a new tab, and the entity tree tab switches to the active entity tab. This is also how editors tend to work in other fields (code editors, art/graphic design software).

Similarly this would also be a nice to have for graphs and other types of assets.

**Comment by simon-ourmachinery** [comment]
@niklas-ourmachinery this might be a great use-case for workspaces?

**Comment by niklas-ourmachinery** [comment]
You can do this already by holding down Ctrl when you double-click on the entity in the asset browser.

Some of these "hold down a key to do something special" things are pretty hard to find though... we should have a better way of documenting them.

**Comment by simon-ourmachinery** [comment]
@niklas-ourmachinery  Maybe we should look at what blender does: Since we have now short cut manager to implement this should be easier.

![image](https://user-images.githubusercontent.com/70658708/128771000-d257dd73-fd83-44e7-9656-30ddb7fb79af.png)

**Comment by LaylBongers** [comment]
In my opinion it makes more sense for this to be the default behavior, as I feel this is what you want to do the majority of the time (as in, open an entity to edit it, then return to the main scene entity).

**Comment by niklas-ourmachinery** [comment]
@simon-ourmachinery We have similar help text at the bottom of the window:

![image](https://user-images.githubusercontent.com/26548514/128777682-9440e2dc-461a-43ee-926b-570f48ae0142.png)

But it needs to be fixed so that the double-click text says "Open in new Window" when Ctrl is held down.

**Comment by simon-ourmachinery** [comment]
Yes I know but we don't show short cuts when you press CTR what can you do with that key? This was something I was more referring to! Since we have a short cut manager we can do this now. It would at least add some more discoverbilty to some shortcuts. But this is off topic.
